README.md

```diff
@@ -18,6 +18,7 @@ Fill Free to use the CheatSheets.
 - [JavaScript](./JavaScript)
 - [SQL](./SQL)
 - [Statistics](./Statistics)
+- [Vim](./Vim)
```

Comment:
Change it to `Code Editor` inside that, add `Vim`

Reply:
Done